Integrations/Whois/Whois.py

```diff
@@ -0,0 +1,8289 @@
+    "hgtv": {
+        "_type": "newgtld",
+        "host": "whois.nic.hgtv"
+    },
+    "hiphop": {
+        "_group": "uniregistry",
+        "_type": "newgtld",
+        "host": "whois.uniregistry.net"
+    },
+    "hisamitsu": {
+        "_group": "gmo",
+        "_type": "newgtld",
+        "host": "whois.nic.gmo"
+    },
+    "hitachi": {
+        "_group": "gmo",
+        "_type": "newgtld",
+        "host": "whois.nic.gmo"
+    },
+    "hiv": {
+        "_group": "uniregistry",
+        "_type": "newgtld",
+        "host": "whois.uniregistry.net"
+    },
+    "hk": {
+        "host": "whois.hkirc.hk"
+    },
+    "inc.hk": {
+        "_group": "udrregistry",
+        "_type": "private",
+        "host": "whois.registry.hk.com"
+    },
+    "ltd.hk": {
+        "_group": "udrregistry",
+        "_type": "private",
+        "host": "whois.registry.hk.com"
+    },
+    "hkt": {
+        "_group": "afiliassrs",
+        "_type": "newgtld",
+        "host": "whois.nic.hkt"
+    },
+    "hm": {
+        "host": "whois.registry.hm"
+    },
+    "hn": {
+        "host": "whois.nic.hn"
+    },
+    "hockey": {
+        "_group": "donuts",
+        "_type": "newgtld",
+        "host": "whois.nic.hockey"
+    },
+    "holdings": {
+        "_group": "donuts",
+        "_type": "newgtld",
+        "host": "whois.nic.holdings"
+    },
+    "holiday": {
+        "_group": "donuts",
+        "_type": "newgtld",
+        "host": "whois.nic.holiday"
+    },
+    "homedepot": {
+        "_group": "afiliassrs",
+        "_type": "newgtld",
+        "host": "whois.nic.homedepot"
+    },
+    "homegoods": {
+        "_type": "newgtld",
+        "adapter": "none"
+    },
+    "homes": {
+        "_group": "donuts",
+        "_type": "newgtld",
+        "host": "whois.afilias-srs.net"
+    },
+    "homesense": {
+        "_type": "newgtld",
+        "adapter": "none"
+    },
+    "honda": {
+        "_type": "newgtld",
+        "host": "whois.nic.honda"
+    },
+    "honeywell": {
+        "_type": "newgtld",
+        "adapter": "none"
+    },
+    "horse": {
+        "_group": "mmregistry",
+        "_type": "newgtld",
+        "host": "whois.nic.horse"
+    },
+    "hospital": {
+        "_group": "donuts",
+        "_type": "newgtld",
+        "host": "whois.nic.hospital"
+    },
+    "host": {
+        "_type": "newgtld",
+        "host": "whois.nic.host"
+    },
+    "hosting": {
+        "_group": "uniregistry",
+        "_type": "newgtld",
+        "host": "whois.uniregistry.net"
+    },
+    "hot": {
+        "_group": "amazonregistry",
+        "_type": "newgtld",
+        "adapter": "none"
+    },
+    "hoteles": {
+        "_type": "newgtld",
+        "adapter": "none"
+    },
+    "hotels": {
+        "_type": "newgtld",
+        "adapter": "none"
+    },
+    "hotmail": {
+        "_type": "newgtld",
+        "adapter": "none"
+    },
+    "house": {
+        "_group": "donuts",
+        "_type": "newgtld",
+        "host": "whois.nic.house"
+ 
```

Comment:
catch value error when using `strftime`
this is relevant also to the following lines.